ksp_plugin/plugin.cpp

```diff
@@ -79,96 +79,108 @@ Plugin::Plugin(Instant const& initial_time,
       current_time_(initial_time),
       history_time_(initial_time) {}
 
-void Plugin::InsertCelestial(
-    Index const celestial_index,
-    GravitationalParameter const& gravitational_parameter,
-    Index const parent_index,
-    RelativeDegreesOfFreedom<AliceSun> const& from_parent) {
-  CHECK(initializing_) << "Celestial bodies should be inserted before the end "
-                       << "of initialization";
-  auto body = std::make_unique<MassiveBody>(gravitational_parameter);
-  LOG(INFO) << "Initial |{orbit.pos, orbit.vel}| for celestial at index "
-            << celestial_index << ": " << from_parent;
-  auto const relative = PlanetariumRotation().Inverse()(from_parent);
-  LOG(INFO) << "In barycentric coordinates: " << relative;
-  DegreesOfFreedom<Barycentric> const& parent_degrees_of_freedom =
-      FindOrDie(*initial_state_, parent_index);
-  DirectlyInsertCelestial(celestial_index,
-                          &parent_index,
-                          parent_degrees_of_freedom + relative,
-                          std::move(body));
-}
+// The three celestial-inserting functions log at the INFO level.  This is not
+// a concern since they are only called at initialization, which should happen
+// once per game.
 
 void Plugin::InsertSun(Index const celestial_index,
```

Comment:
It's unclear if this function should still exist. It's now only usable for hierarchical initialization. Maybe the parent index should be optional in the hierarchical case and this function can go away. Leave a to-do?